src/components/30-organisms/modal/child.scss

```diff
@@ -0,0 +1,19 @@
+axa-modal[forced] > * {
+  &:first-child {
+    // !important reason: An axa-heading child element sets margins on the host
+    // element. This can only be overridden by !important.
+    // This is harmless because: There is no scenario where it would make sense
+    // for the first element to have any margin-top, except if they work with
+    // negative margins, which is an anti-pattern.
+    margin-top: 0 !important;
+  }
+
+  &:last-child {
+    // !important reason: An axa-heading child element sets margins on the host
+    // element. This can only be overridden by !important.
+    // This is harmless because: There is no scenario where it would make sense
+    // for the first element to have any margin-top, except if they work with
+    // negative margins, which is an anti-pattern.
+    margin-bottom: 0 !important;
```

Comment:
This do not have any effect at SB example?

Reply:
Yeah but does that matter? The button does not have any margins by default. That is ok too if you ask me. If it would have, this css property would erase it.

Reply:
This is a fix for something wo do not know if it will occur in future. I would just set margin-top.